src/libstd/time/duration.rs

```diff
@@ -53,6 +53,10 @@ pub struct Duration {
 }
 
 impl Duration {
+    /// Constant zero duration for use within libstd
+    #[allow(dead_code)]
+    pub(crate) const ZERO: Duration = Duration { secs: 0, nanos: 0};
```

Comment:
Could this perhaps leverage https://github.com/rust-lang/rust/pull/47300 instead of adding a new constant?